support-frontend/assets/helpers/internationalisation/localCurrencyCountry.js

```diff
@@ -24,6 +25,15 @@ export const localCurrencyCountries: {
         defaultAmount: 50,
       },
     },
+    config: {
+      ONE_OFF: {
```

Comment:
I'm surprised flow doesn't require monthly/annual...

Reply:
It didn't complain about it, but it _did_ about the `default` value here. 🤷🏼

Reply:
It's really a flaw in flow because the following can be unsafe:
`localCurrencyCountry.config[contributionType].min`
There's a thread about it here: https://github.com/facebook/flow/issues/5276

When we migrate to TS this will not be possible! I guess we can live with it until then as we're not planning to do the work for recurring any time soon